AutocompleteClientTests/FW/Logic/Worker/ConstructorIOIntegrationTests.swift

```diff
@@ -11,7 +11,7 @@ import ConstructorAutocomplete
 
 // swiftlint:disable type_body_length
```

Comment:
Looks like swift lint is throwing an error because the file is longer than 1,000 lines now 😔 

We'll want to update this comment here to be:

`// swiftlint:disable type_body_length file_length`